starter-code/index.js

```diff
@@ -1,10 +1,27 @@
-var SortedList = function() {};
+var SortedList = function() {
+  this.length = [];
+};
 
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+SortedList.prototype.add = function(item) {
+  this.length.push(item);
+  this.length.sort((a, b) => a - b);
+}
 
+SortedList.prototype.get = function(pos) {
+  return this.length[pos - 1];
+}
```

Comment:
Os falla parte del test porque la función debe estar dividida en dos. En caso de no existir la posición en el array, ej. porque buscas la posición 20 en un array de 2 elementos, debe devolver un string que diga 'Out of bounds'(o algo así).